src/tests/multihost/alltests/test_misc.py

```diff
@@ -161,7 +161,40 @@ def test_0003_sssd_crashes_after_update(self, multihost,
         assert status == "PASS"
 
     @pytest.mark.tier1
-    def test_0004_sssd_api_conf(self, multihost, backupsssdconf):
+    def test_0004_client_timeout(self, multihost, backupsssdconf):
+        """
+        :Title: misc: KCM: Increase client idle
+        timeout to 5 minutes
+
+        @bugzilla:
+        https://bugzilla.redhat.com/show_bug.cgi?id=1884205
+        """
+        client = sssdTools(multihost.client[0])
+        domain_params = {'debug_level': '9'}
+        client.sssd_conf('kcm', domain_params)
+        multihost.client[0].service_sssd('restart')
+        user = 'foo1@example1'
+        client = pexpect_ssh(multihost.client[0].sys_hostname, user,
+                             'Secret123', debug=False)
+        client.login(login_timeout=30, sync_multiplier=5,
+                     auto_prompt_reset=False)
+        sssdTools(multihost.client[0]).clear_sssd_cache()
+        multihost.client[0].run_command("systemctl restart sssd-kcm")
+        multihost.client[0].run_command("> /var/log/sssd/sssd_kcm.log")
+        start_time = time.time()
+        multihost.client[0].run_command("kinit foo1 <&- & ")
+        end_time = time.time()
+        assert end_time - start_time >= 300
+        grep_cmd = multihost.client[0].run_command("grep"
+                                                   " 'Terminated"
+                                                   " client'"
+                                                   " /var/log/sssd/"
+                                                   "sssd_kcm.log")
+        assert 'Terminated client' in grep_cmd.stdout_text
```

Comment:
keep assert statement as the last statement because if code fails, it won't check the client.logout basically the next statement after assert.